demostat/templates/demostat/demo_detail.html

```diff
@@ -40,12 +43,23 @@ <h5 class="card-title">Was?</h5>
     </div>
   </div>
   {% if demo.link_set.all %}
+  {% if demo.location.lat and demo.location.lon or demo.demo_set.all %}
   <div class="list-group list-group-horizontal-md text-center">
     {% for link in demo.link_set.all|slice:":4" %}
     <a href="{{ link.url }}" class="list-group-item flex-fill" target="_blank">{{ link.title }}</a>
     {% endfor %}
   </div>
   {% endif %}
+  {% endif %}
+  {% if demo.demo_set.all %}
+  <div class="row">
+    {% for de in demo.demo_set.all %}
+    <div class="col-md-4">
+      {% include "demostat/components/demo_card.html" with demo=de %}
```

Comment:
```suggestion
      {% if not de.id == demo.id %}
        {% include "demostat/components/demo_card.html" with demo=de %}
      {% endif %}
```

Wenn eine Demo auf sich selbst verweist sollte diese nicht so angezeigt werden

Reply:
Keine dumme Idee. An dieser Stelle ist jedoch die Umsetzung fehlerhaft. Die Bedingung muss noch vor `<div class="col-md-4">` gestellt werden.

Reply:
Ich werde das entsprechend in einem Commit ausbessern.